2023-06-04 00:00:36.221 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:03:17.573 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:04:11.933 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:04:33.729 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:09:55.876 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:21:22.489 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:25:21.371 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:25:30.805 +02:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unsupported handler method return type 'ReversiApp.Areas.Identity.Pages.Account.RegisterModel+PasswordScore'.
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.CreateHandlerMethod(HandlerMethodDescriptor handlerDescriptor)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.CreateExecutor(HandlerMethodDescriptor handlerDescriptor)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvokerProvider.GetHandlerExecutors(CompiledPageActionDescriptor actionDescriptor)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvokerProvider.CreateCacheEntry(ActionInvokerProviderContext context, FilterItem[] cachedFilters)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvokerProvider.OnProvidersExecuting(ActionInvokerProviderContext context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionInvokerFactory.CreateInvoker(ActionContext actionContext)
   at Microsoft.AspNetCore.Mvc.Routing.ActionEndpointFactory.<>c__DisplayClass7_0.<CreateRequestDelegate>b__0(HttpContext context)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2023-06-04 00:26:29.612 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:29:09.354 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:32:40.234 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:33:31.731 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:34:10.403 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:34:50.267 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:35:22.911 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:36:46.654 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:37:22.613 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:37:45.844 +02:00 [ERR] An unhandled exception has occurred while executing the request.
System.NullReferenceException: Object reference not set to an instance of an object.
   at ReversiApp.Areas.Identity.Pages.Account.RegisterModel.CheckStrength(String password) in D:\code\git\jaar2-reversiServer\ReversiApp\Areas\Identity\Pages\Account\Register.cshtml.cs:line 104
   at ReversiApp.Areas.Identity.Pages.Account.RegisterModel.OnGetFindUser(String password) in D:\code\git\jaar2-reversiServer\ReversiApp\Areas\Identity\Pages\Account\Register.cshtml.cs:line 126
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.ActionResultHandlerMethod.Execute(Object receiver, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2023-06-04 00:37:45.844 +02:00 [ERR] An unhandled exception has occurred while executing the request.
System.NullReferenceException: Object reference not set to an instance of an object.
   at ReversiApp.Areas.Identity.Pages.Account.RegisterModel.CheckStrength(String password) in D:\code\git\jaar2-reversiServer\ReversiApp\Areas\Identity\Pages\Account\Register.cshtml.cs:line 104
   at ReversiApp.Areas.Identity.Pages.Account.RegisterModel.OnGetFindUser(String password) in D:\code\git\jaar2-reversiServer\ReversiApp\Areas\Identity\Pages\Account\Register.cshtml.cs:line 126
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.ActionResultHandlerMethod.Execute(Object receiver, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2023-06-04 00:41:13.646 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:41:51.335 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:44:28.465 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:45:54.690 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:47:02.638 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:48:51.831 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 00:54:54.962 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:00:08.390 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:03:57.729 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:07:24.401 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:07:39.467 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:10:55.989 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:11:42.030 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:14:30.432 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:14:52.320 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:16:10.530 +02:00 [ERR] An unhandled exception has occurred while executing the request.
System.NullReferenceException: Object reference not set to an instance of an object.
   at ReversiApp.Areas.Identity.Pages.Account.RegisterModel.CheckStrength(String password) in D:\code\git\jaar2-reversiServer\ReversiApp\Areas\Identity\Pages\Account\Register.cshtml.cs:line 106
   at ReversiApp.Areas.Identity.Pages.Account.RegisterModel.OnGetFindUser(String password) in D:\code\git\jaar2-reversiServer\ReversiApp\Areas\Identity\Pages\Account\Register.cshtml.cs:line 127
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.ActionResultHandlerMethod.Execute(Object receiver, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2023-06-04 01:16:31.604 +02:00 [ERR] An unhandled exception has occurred while executing the request.
System.NullReferenceException: Object reference not set to an instance of an object.
   at ReversiApp.Areas.Identity.Pages.Account.RegisterModel.CheckStrength(String password) in D:\code\git\jaar2-reversiServer\ReversiApp\Areas\Identity\Pages\Account\Register.cshtml.cs:line 106
   at ReversiApp.Areas.Identity.Pages.Account.RegisterModel.OnGetFindUser(String password) in D:\code\git\jaar2-reversiServer\ReversiApp\Areas\Identity\Pages\Account\Register.cshtml.cs:line 127
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.ActionResultHandlerMethod.Execute(Object receiver, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2023-06-04 01:17:46.825 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:19:14.635 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:23:53.180 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:24:41.974 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:25:25.848 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:25:58.358 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:27:39.842 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:29:09.018 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:34:00.118 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:35:20.419 +02:00 [WRN] User 805f1a0d-3914-48a9-9fbd-09b623cd9ea5 password validation failed: PasswordRequiresDigit;PasswordRequiresLower;PasswordRequiresUpper.
2023-06-04 01:43:53.602 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 01:50:57.605 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 12:41:20.292 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 12:44:18.867 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 12:50:23.383 +02:00 [WRN] User d2051f4f-d3bf-4ae9-afca-0a2f6cc6d5d7 password validation failed: PasswordTooShort.
2023-06-04 12:57:51.133 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 12:58:15.263 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 13:04:51.530 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 13:05:53.340 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 13:06:36.701 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 13:07:11.968 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 19:07:18.883 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 19:19:58.610 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 19:20:42.793 +02:00 [WRN] User 75bf682d-74e8-4ca0-9211-121f41abd45c password validation failed: PasswordTooShort.
2023-06-04 19:20:57.162 +02:00 [WRN] User 5211f0c5-7f0b-4de2-8ac9-7ec3a066fc3f validation failed: InvalidUserName.
2023-06-04 19:43:05.970 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 20:56:19.133 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 21:00:07.470 +02:00 [FTL] Een gebruiker met de naam  kon niet worden gevonden!
System.ArgumentNullException: Value cannot be null. (Parameter 'userName')
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at ReversiApp.Controllers.SpelController.Index() in D:\code\git\jaar2-reversiServer\ReversiApp\Controllers\SpelController.cs:line 37
2023-06-04 21:03:18.821 +02:00 [WRN] This test message can be safely ignored!
2023-06-04 23:11:21.752 +02:00 [WRN] This test message can be safely ignored!
